.github/workflows/release.yml

```diff
@@ -26,6 +26,12 @@ jobs:
 
       - uses: actions/checkout@v2
 
+      - name: Check consistency of deployment yamls
+        if: ${{ ! github.event.release.prerelease }} # On pre-releases, the -pre tag is added making this more complex
```

Comment:
why we are not checking this on prerelease?

Reply:
isn't better to catch the essero as soon as possible?

Reply:
I tweaked it to strip the `-pre` suffix in the version check. It is not ideal, but should work without the need to change the yaml between prereleases.